nmigen/test/test_hdl_ast.py

```diff
@@ -908,14 +908,17 @@ def lower(self):
 
 
 class UserValueTestCase(FHDLTestCase):
-    def test_shape(self):
-        uv = MockUserValue(1)
+    def test_shape(self, lower_to=int):
+        uv = MockUserValue(lower_to(1))
         self.assertEqual(uv.shape(), unsigned(1))
         self.assertIsInstance(uv.shape(), Shape)
-        uv.lowered = 2
+        uv.lowered = lower_to(2)
         self.assertEqual(uv.shape(), unsigned(1))
         self.assertEqual(uv.lower_count, 1)
```

Comment:
This test is a bit hard to understand--could you duplicate the code? It's not much and it makes it much more clear what's going on.

Reply:
k